frontend/src/metabase/visualizations/shared/components/RowChart/RowChart.tsx

```diff
@@ -48,7 +48,7 @@ export interface RowChartProps<TDatum> {
 
   tickFormatters?: ChartTicksFormatters;
   labelsFormatter?: (value: NumberValue) => string;
-  measureText: TextMeasurer;
+  measureText: TextWidthMeasurer;
```

Comment:
we changed the type here, but never the function that was passed, causing https://github.com/metabase/metabase/issues/32044

Reply:
why didn't typescript catch this?

Reply:
The visualization parent component is still javascript